competitions/template/ansible/inv/prod/group_vars/all/secret.tmpl

```diff
@@ -0,0 +1,11 @@
+---
+# mysql has no root password on this server
+mysql_root_password:
+
+db_password: <PASSWORD>
+mediawiki_admin_password: <PASSWORD>
+mediawiki_mwlib_password: <PASSWORDWITHNOSPACES>
```

Comment:
"PASSWORD_WITH_NO_SPACES".  I know that seems self-contradictory at first, but it's not, really, and it's easier to read :-).